Migration-v11.md

```diff
@@ -0,0 +1,89 @@
+# Migration Guide for v11 (Brand Refresh)
+
+## Migration with Codemod
+
+**Step1:** Install this version of `@razorpay/blade`.
+
+**Step2:** Execute the codemod using the following command:
+
+> Need help? Check out [jscodeshift docs](https://github.com/facebook/jscodeshift) for CLI usage tips.
+
+```sh
+npx jscodeshift ./YOUR_DIR --extensions=tsx,ts,jsx,js -t ./node_modules/@razorpay/blade/codemods/brand-refresh/transformers/index.ts --ignore-pattern="**/node_modules/**"
+```
+
+> [!WARNING]
+>
+> While this codemod covers most cases, it's always good to double-check and test your code to catch any missed nuances.
+
+### 🚧 Watch Out for Limitations & Edge Cases
+
+> [!IMPORTANT]
+>
+> There might be some situations where the codemod falls short. If you encounter errors, handle those cases manually for a seamless transition. Happy coding!
+
```

Comment:
mention that there will be cases where the script will add that weird string and people shall talk to designs and fix those manually

Reply:
added